src/features/Chat/chatSlice.ts

```diff
@@ -238,6 +238,16 @@ export const selectChats = createSelector(
   },
 );
 
+export const selectHasUnreadMessages = createSelector(
+  selectChatState,
+  ({ chats }) =>
+    Object.values(chats)
+      .filter(chat => chat.status === 'ok')
+      .map(chat => chat.messages)
+      .flat()
+      .filter(message => !message.opened).length > 0,
```

Comment:
Great! Just fyi:

 currently in the mobile app we also show a unseenDot for the archived messages.
I am not sure if we want the same behaviour in the new app, probably.

Also nitpick: i would prefer to use .some() instead of filter and comparing to length.

Anyways great job! (No need to change these now,

Reply:
Refactored the filter + length to some!